packages/liveblocks-react/src/room.tsx

```diff
@@ -538,6 +539,7 @@ function makeRoomContextBundle<
     // context consistent internally.
     return (
       <LiveblocksProviderWithClient client={client} allowNesting>
+        {/* Why type error? */}
```

Comment:
@FlowFlorent What's this? There should not be any type error, so lmk if this is causing issues for you.

Reply:
FYI: I removed the comment to tidy up this PR